docs/howto/openstack/neutron/del-network.md

```diff
@@ -0,0 +1,553 @@
+# Deleting networks
+
+Deleting a network in {{brand}} may sound like a pretty straightforward 
+task --- and it is. It's just that before deleting a network, there are 
+some steps we almost always need to take. In what follows we show, step 
+by step and through specific examples, how we delete networks using 
+either the {{gui}} or the OpenStack CLI.
+
+## Prerequisites
+
+Whether you choose to work from the {{gui}} or with the OpenStack CLI, 
+you need to [have an account](/howto/getting-started/create-account) in 
+{{brand}}. Additionally, to use the OpenStack CLI, make sure to [enable 
+it](/howto/getting-started/enable-openstack-cli) for the region you 
+will be working in.
+
+## Selecting a network
+
+Unless you already have the ID or know the name of the network you wish 
+to delete, you may first list all available networks. To do so from the 
+{{gui}}, fire up your favorite web browser, navigate to the [Cleura 
+Cloud](https://{{gui_domain}}) start page, and log into your {{brand}} 
+account. On the other hand, if you prefer to work with the OpenStack 
+CLI, please do not forget to [source the RC 
+file](/howto/getting-started/enable-openstack-cli) first.
```

Comment:
I think everything after "you may first list all available networks" belongs under the subsequent content tabs.